test/unit/controllers/HasExistingEoriControllerSpec.scala

```diff
@@ -220,20 +220,11 @@ class HasExistingEoriControllerSpec extends ControllerSpec with BeforeAndAfterEa
     }
   }
 
-  "Has Existing EORI Controller enrol confirmation page" should {
```

Comment:
We no longer need this test, we do not pass an Eori through to the confirmation page anymore